aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -183,6 +190,15 @@ TORCH_IMPL_FUNC(polygamma_out)
   polygamma_stub(device_type(), *this, n);
 }
 
+TORCH_IMPL_FUNC(signbit_out) (const Tensor& self, const Tensor& result) {
+  at::native::resize_output(result, self.sizes());
```

Comment:
This is unnecessary, `TORCH_META_FUNC` guarantees that the result is sized correctly